dataflux_pytorch/benchmark/llama-70b/load.py

```diff
@@ -0,0 +1,37 @@
+import time
+
+import torch
+
+from dataflux_pytorch import dataflux_checkpoint
+
+start_time = time.time()
+ckpt = dataflux_checkpoint.DatafluxCheckpoint(project_name="<PROJECT_NAME>",
+                                              bucket_name="<BUCKET_NAME")
+CKPT_PATH = "llama2/llama2-70b-hf/"
+LOAD_COUNT = 10
+files = [
```

Comment:
nit: we can simplify generation of file names ?

Reply:
I just want to leave it explicit since this matches the exact format of the files from huggingface. There are multiple sets of files (such as the safe-tensor set)